agent/operational_incidents.py

```
@@ -0,0 +1,500 @@
+def _schedule(coro: Any) -> None:
+    """Run *coro* without blocking the caller, from any thread.
+
+    The caller is a logging handler, which may be on the event loop thread, a
+    worker thread, or a process with no loop at all.
+    """
+    try:
+        loop = asyncio.get_running_loop()
+    except RuntimeError:
+        threading.Thread(target=asyncio.run, args=(coro,), daemon=True).start()
+        return
+    loop.create_task(coro)
```

Ruff (0.16.1)

[warning] Store a reference to the return value of `loop.create_task`

(RUF006)